Dockerfile

```diff
@@ -1,26 +1,24 @@
-FROM python:3-slim AS builder
-ENV MARK="9.9.0"
-
+FROM python:3.11-slim-bookworm AS builder
```

Comment:
I would use something less specific as tag in order to not have to change the code every time, but just rebuil the image like `3.11-slim`